tap_rest_api_msdk/streams.py

```diff
@@ -287,7 +287,7 @@ def get_new_paginator(self):
         elif (
             self.pagination_request_style == "simple_header_paginator"
         ):  # Example Gitlab.com
-            return SimpleHeaderPaginator("X-Next-Page")
+            return SimpleHeaderPaginator(self.next_page_token_jsonpath)
```

Comment:
This could be a breaking change. `tap.py` indicates that `next_page_token_path` defaults to `Null`. Although the description string indicates otherwise, which means that that description should be updated.

It would be safer if this logic were updated to something similar to the way it's handled in the SDK:
```
        if self.next_page_token_jsonpath:
            return JSONPathPaginator(self.next_page_token_jsonpath)

        return SimpleHeaderPaginator("X-Next-Page")
```

This would also help avoid a breaking change.

Reply:
Changed according to your note.